### DISCUSSION TITLE
Regarding API of online validator

### DISCUSSION THREAD

**Discussion by SoumithMamidyala** [CMSgov/hospital-price-transparency#156] [Q&A] [answered]
Hi, can I know how I get the API for online validator.

**Comment by RidenourK** [comment, marked as answer]
@SoumithMamidyala. Thank you for your question. 

The online validator doesn’t have api endpoints that a client can access from the network. It provides a web form interface to submit MRF files, but the validation is performed within the user’s browser. The repository at https://github.com/CMSgov/hpt-validator contains the validation code that is used by the online validator.
